README.md

```diff
@@ -1,2 +1,28 @@
 # Portfolio-Moblie-Skeleton-Version
 Building mobile skeleton version of personal website.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live demo
+
+[LiveDemo](http://127.0.0.1:5501/index.html)
```

Comment:
- [ ] This will work when user use vs code and live extension, otherwise not. If you want to include this, Please write the steps on how to clone and open the folder and the run the project.

Hint
```
git clone https://github.com/Christianib003/Portfolio-Moblie-Skeleton-Version.git

cd Portfolio-Moblie-Skeleton-Version

open index.html
``` 
to run the application